src/microbe_stage/MicrobeInternalCalculations.cs

```diff
@@ -170,6 +170,46 @@ public static float RotationSpeedToUserReadableNumber(float rawSpeed)
         return rawSpeed * 500;
     }
 
+    public static float CalculateDigestionSpeed(float enzymeCount)
```

Comment:
As far as I can tell enzyme count is an int in this file so why is the param type a float?